RecoPixelVertexing/PixelTriplets/plugins/CAHitQuadrupletGeneratorGPU.cc

```diff
@@ -23,6 +26,33 @@ namespace {
     return x * x;
   }
 
+  CAHitQuadrupletGeneratorKernels::QualityCuts makeQualityCuts(edm::ParameterSet const& pset) {
+    auto coeff = pset.getParameter<std::vector<double>>("chi2Coeff");
+    if (coeff.size() != 4) {
+      throw edm::Exception(edm::errors::Configuration, "CAHitQuadrupletGeneratorGPU.trackQualityCuts.chi2Coeff must have 4 elements");
+    }
+    return CAHitQuadrupletGeneratorKernels::QualityCuts {
+      // polynomial coefficients for the pT-dependent chi2 cut
+      { (float) coeff[0], (float) coeff[1], (float) coeff[2], (float) coeff[3] },
+      // max pT used to detrmine the chi2 cut
+      (float) pset.getParameter<double>("chi2MaxPt"),
+      // chi2 scale factor: 30 for broken line fit, 45 for Riemann fit
```

Comment:
Would it be worth to add the comment on values for broken line and Riemann to the `setComment()` in `fillDescriptions()`?

Reply:
I was going to add a cfi with the configuration for the Riemann fit somewhere in the release, but sure, I can add the `30` vs `45` in the comment as well.

Reply:
done